deploy/instance/remote-scripts/undeploy.sh

```
@@ -0,0 +1,2 @@
+cd /root/saf-2025/deploy/instance/remote-assets
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)